sdk/src/math/orders.ts

```diff
@@ -283,3 +283,7 @@ export function isTriggered(order: Order): boolean {
 		'triggeredBelow',
 	]);
 }
+
+export function isRestingLimitOrder(order: Order, slot: number): boolean {
+	return order.postOnly || isAuctionComplete(order, slot);
```

Comment:
i see this is used only for the limit order lists, but since it's in the function name, should you also check that the `OrderType` is a limit order to prevent incorrect usage?